ext/standard/basic_functions.stub.php

```diff
@@ -275,6 +275,167 @@ function base64_encode(string $str): string {}
 /** @return string|false */
 function base64_decode(string $str, bool $strict = false) {}
 
+/* basic_functions.c */
+
+/** @return mixed */
+function constant(string $name) {}
+
+/** @return int|false */
+function ip2long(string $ip_address) {}
+
+/** @return string|false */
+function long2ip(int $proper_address) {}
+
+/** @return string|array|false */
+function getenv(string $variable = UNKNOWN, bool $local_only = false) {}
+
+#ifdef HAVE_PUTENV
+function putenv(string $setting): bool {}
+#endif
+
+/** @return array|false */
+function getopt(string $options, array $longopts = UNKNOWN, &$optind = NULL) {}
+
+function flush(): void {}
+
+function sleep(int $seconds): ?int {}
```

Comment:
Wow, that's weird. Looks like PHP_SLEEP_NON_VOID is always set though, probably we can remove this oddity.